app/controllers/rooms_controller.rb

```diff
@@ -8,11 +8,9 @@ def new
   end
   
   def create
-    p params
```

Comment:
Logging values like this is useful for development, but should not clog up the production logs.